Build/AzureDevOps/azure-pipelines.yml

```diff
@@ -62,7 +62,65 @@ jobs:
         testResultsFiles: built\Out\windows_$(buildPlatform)\$(buildConfiguration)\GLTFSDK.Test\GLTFSDK.Test.log
       condition: and(succeeded(), in(variables['buildPlatform'], 'Win32', 'x64'))
 
-  - job: WindowsVcPkg
+  - job: Windows_VS2019
```

Comment:
We probably should start factoring this out a bit as to not repeat the same thing multiple times.

Reply:
Talked offline about this.